lib/js/components/TagList/TagList.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+	<div
+		:class="{
+			tagList: true,
+			'-neutral': color === TAG_LIST_COLORS.NEUTRAL,
+			'-neutralHeavy': color === TAG_LIST_COLORS.NEUTRAL_HEAVY,
+			'-xSmall': size === TAG_LIST_SIZES.X_SMALL,
+			'-small': size === TAG_LIST_SIZES.SMALL,
+			'-medium': size === TAG_LIST_SIZES.MEDIUM,
+		}"
+	>
+		<icon class="tagList__icon" :icon="ICONS.FA_TAGS" :size="ICON_SIZES.X_SMALL" />
+		<div>{{ tagsConcatenated }}</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+
+.tagList {
+	$root: &;
+
+	align-items: flex-start;
+	display: flex;
```

Comment:
`display: flex` ? Did you try to put this component next to some text? IMO, it looks more like `inline-flex` component

Reply:
Why? We won't be using it in a middle of paragraph. It's a block element that is shown below other elements.

Reply:
maybe it's a question to UX, for me it looks like possible scenario to put it in the text

Reply:
https://bethinkteam.slack.com/archives/C012R8RM3A4/p1665126117257579

Reply:
We've agreed to not change it.